index.js

```diff
@@ -9,7 +9,7 @@ const createServer = (options) => {
   const path = options.path || process.cwd()
   const app = express()
 
-  app.use(express.json())
+  app.use(express.json({ type: '*/*' }))
```

Comment:
Go catch!! However, I think we need to use this middleware `app.use(express.urlencoded())` because the value `*/*` is just bypassing the content type but the middleware tries to parse it as a JSON.

Reply:
I think this is correct because the client sends JSON body even when content-type is `application/x-www-form-urlencoded`. See https://github.com/algolia/algoliasearch-client-javascript/issues/667